app/wallet/components/deposit-options.tsx

```diff
@@ -56,9 +61,13 @@ export default function DepositOptions({ onClose, onDirectDepositClick }: Deposi
           <Building2 className="w-6 h-6 text-muted-foreground" />
         </div>
         <div className="flex-1">
-          <h3 className="text-base font-bold text-black leading-6 mb-1">Direct deposit</h3>
+          <h3 className="text-base font-bold text-black leading-6 mb-1">
+            {operation === "DEPOSIT" ? "Direct deposit" : "Direct withdrawal"}
+          </h3>
           <p className="text-muted-foreground text-sm font-normal leading-[22px]">
-            Deposit funds directly from your bank account, e-wallet, or other payment methods.
+            {operation === "DEPOSIT"
+              ? "Deposit funds directly from your bank account, e-wallet, or other payment methods."
+              : "Withdraw funds directly to your bank account, e-wallet, or other payment methods."}
```

Comment:
there's a lot of conditions here, i think it would be better to create a separate component for withdrawal